docs/src/index.md

```diff
@@ -31,7 +31,8 @@ The [source code repository is in Github](https://github.com/cloudnative-pg/clou
 
 ## Supported Kubernetes distributions
 
-CloudNativePG requires Kubernetes 1.22 or higher. For more information, please
+CloudNativePG requires Kubernetes 1.23 or higher for the latest 1.18 release.
+Previous releases can also support Kubernetes 1.22. For more information, please
```

Comment:
I do not like this change. It is not clear and not generic enough. I would revert this change and restore the previous version.